### PR TITLE
[Catalog] Fix CardSelectionModeActivity sample

### PR DESCRIPTION
**Pull request by pubiqq**
Resolves https://github.com/material-components/material-components-android/issues/2716

### CODE REVIEW

File: catalog/java/io/material/catalog/card/SelectableCardsAdapter.java [hunk 1]
```diff
@@ -99,13 +99,6 @@ private void bind(Item item, int position) {
       if (selectionTracker != null) {
         bindSelectedState();
       }
-      // Set an OnLongClickListener for accessibility
-      materialCardView.setOnLongClickListener(
-          v -> {
-            selectionTracker.setItemsSelected(
```
Comment (drchen) [inline catalog/java/io/material/catalog/card/SelectableCardsAdapter.java:105]: I think the right fix should be adding an if check instead:

if (materialCardView.isChecked()) {
  selectionTracker.xxxxxx....
  return true;
}
return false;

Comment (pubiqq) [reply catalog/java/io/material/catalog/card/SelectableCardsAdapter.java:105]: `SelectionTracker` manages the list item selections on its own, it shouldn't need additional `onClickListener`.

**Comment by drchen** [comment]
The original commit that added this solution/workaround: https://github.com/material-components/material-components-android/commit/a49fa6ab36276a762d46d9f716b4ef8ff5947907

If I understand correctly, `SelectionTracker` is using `RecyclerView.OnItemTouchListener` to grab click/long click events. Therefore if the accessibility framework directly send click/long click events, `SelectionTracker` won't be able to catch them, and that's why we need the workaround at the beginning and we need to keep the behavior.

**Comment by drchen** [comment]
Paul, can you take a look?

**Comment by pubiqq** [comment]
boop

**Comment by pubiqq** [comment]
any news?

**Comment by pubiqq** [comment]
any news?

**Comment by paulfthomas** [comment]
This just reverts the original change that was made to make (attempt...) the card selection accessible, would be great to find a proper fix instead of just removing it.

**Comment by pubiqq** [comment]
A proper fix can only be provided by `androidx.recyclerview:recyclerview-selection` developers. 
The original solution is correct from a logical point of view, but doesn't work only with Talkback due to some problem with `recyclerview-selection`. The current solution is a hack and doesn't work correctly at all (neither in normal mode nor in Talkback). That's why I think the current change needs to be reverted.

**Comment by pubiqq** [comment]
bump

**Comment by pubiqq** [comment]
bump

**Comment by pubiqq** [comment]
bump

**Comment by pubiqq** [comment]
bump

**Pull request closed without merging**